course7/homeworks/nadinasovaiala/assign1.cc

```diff
@@ -0,0 +1,12 @@
+#include <iostream>
+#include <boost/filesystem.hpp>
+
+using namespace boost::filesystem;
```

Comment:
Don't do it.
If you think ```boost::filesystem``` is too long, create a shorter alias:
```
namespace bfs = boost::filesystem;
```
but don't drop the qualification altogether.